packages/image_picker/ios/Classes/ImagePickerPlugin.m

```diff
@@ -241,6 +241,16 @@ - (void)imagePickerController:(UIImagePickerController *)picker
     return;
   }
   if (videoURL != nil) {
+    if (@available(iOS 13.0, *)) {
+      NSString *fileName = [videoURL lastPathComponent];
+      NSURL *destination =
+          [NSURL fileURLWithPath:[NSTemporaryDirectory() stringByAppendingPathComponent:fileName]];
+
+      if ([[NSFileManager defaultManager] isReadableFileAtPath:[videoURL path]]) {
+        [[NSFileManager defaultManager] copyItemAtURL:videoURL toURL:destination error:nil];
```

Comment:
At the first glance, I think we should record the error and return it if copy is not successful.
```objc
NSError *error;
[[NSFileManager defaultManager] copyItemAtURL:videoURL toURL:destination error:&error];
if (error) {
   self.result([FlutterError errorWithCode:@"flutter_image_picker_copy_video_error"...);
} else {
   self.result(videoURL.path);
}

```

Reply:
Added the error handler like suggested.